Tests/TuistKitTests/Services/Inspect/InspectBuildCommandServiceTests.swift

```diff
@@ -6,13 +6,15 @@ import TuistCore
 import TuistLoader
 import TuistServer
 import TuistSupport
+import TuistSupportTesting
 import XcodeGraph
 
 @testable import TuistKit
 
+@Suite(.withMockedEnvironment)
```

Comment:
😬 You made the same assumption that I did... This is not run for every test, but once for all, which means we have an environment state shared among all the tests. You'll have to do `@Test(.withMockedEnvironment)` instead.